utilities/cli/tests/test_cli.py

```diff
@@ -218,9 +228,12 @@ def test_lint_fix_command(self):
         args = self.cli.parser.parse_args("lint --fix".split())
         try:
             args.func(args)
+        except FileNotFoundError as e:
+            self.assertIn("ruff", str(e))  # if not installed, it complains about ruff
```

Comment:
Could you explain this check? Do we really want to test linting without all lint dependencies installed?